uswds/uswds-sandbox
# [Test] USWDS-Sandbox: Use uswds 2.14.0 and compile 1.2.0

## Summary
Created an implementation that both uses USWDS 2 and @uswds/compile  1.2.0 so that we can test the compile release with both major versions of the design system.

> [!important]
> This PR is for testing purposes only and should not be merged into main.

> [!Tip]
> We could use this PR to test future releases of uswds compile with uswds v2. We just need to update the version of `@uswds/compile in package.json` (or just do a clean install if the release is in `develop`.

## Major changes
1. Replaced `@uswds/uswds 3.8.1` dependency with `uswds 2.14.0`
1. Copied over the `/_sass` directory from `develop` into a new `src/sass` directory
1. Updated gulpfile  to use `uswds.settings.version = 2;` and `uswds.paths.dist.theme = "./src/sass";`
1. Replaced `uswds/compile 1.1.0` dependency with uswds compile `develop` branch for testing

## Testing and review
1. Open this branch locally
2. Run a clean npm install
3. Run `npx gulp init`
4. Run gulp tasks from `gulpfile.js` like `npx gulp watchSass`, `npx gulp compileIcons` and confirm there are no errors
